mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -174,6 +174,26 @@ public EventBus eventBus3() {
         return eventBus3;
     }
 
+    @Override
+    public EventDeadLetters deadLetter() {
+        return null;
```

Comment:
tthrow new NotImplementedException("We did not plugged yet the DeadLetter in RabbitMQ code...")